cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -756,11 +695,12 @@ def kick_api_server(tls):
     if data_changed('cert', tls.get_server_cert()):
         # certificate changed, so restart the api server
         hookenv.log("Certificate information changed, restarting api server")
-        restart_apiserver()
+        service_restart('snap.kube-apiserver.daemon')
```

Comment:
Old status: `Restarting kube-apiserver`
New status: `Restarting snap.kube-apiserver.daemon service`

I was going to complain about this change - seems noisy, and a bit alien to anyone who's not familiar with CDK. But thinking about it more, I kind of like the idea that the charm status will now introduce users to the idea of snap.*.daemon services so they might have a better idea where to look for errors and whatnot.

So I'm vaguely +1 to this change.

Reply:
I was somewhat torn here as well. On one hand I could change restart_apiserver to use service_restart, but on the other hand it seems more straight-forward what was actually happening. There are no side-effects of service_restart, but restart_apiserver could do other things and might take some jumping around to find. I'm happy either way in the end.